src/bpmncwpverify/typechecking.py

```diff
@@ -0,0 +1,54 @@
+from returns.result import Failure, Result, Success
```

Comment:
Should I be using this once I pull it down for my expression type checker?

Reply:
Yes. It computes the types for literals, and it provides the type-check for assignment compatibility. It needs to be expanded with other compatibility checks: logical connectives and relational operators.